hack/e2e/run-rp-and-e2e.sh

```diff
@@ -69,8 +69,17 @@ clean_e2e_db(){
 
 # TODO: CLUSTER and is also recalculated in multiple places
 # in the billing pipelines :-(
-export CLUSTER="v4-e2e-V$BUILD_BUILDID-$LOCATION"
-export DATABASE_NAME="v4-e2e-V$BUILD_BUILDID-$LOCATION"
+
+
+# if LOCAL_E2E is set, set the value with the local test names
+# If it it not set, it defaults to the build ID 
+if [ -z "${LOCAL_E2E}" ] ; then 
+    export CLUSTER="v4-e2e-V$BUILD_BUILDID-$LOCATION"
+    export DATABASE_NAME="v4-e2e-V$BUILD_BUILDID-$LOCATION"
+else 
+    export CLUSTER="$USER-local-e2e"
+    export DATABASE_NAME="$USER-local-e2e"
+fi 
```

Comment:
This name might not be optimal, I couldn't come up with a better name though. Please feel free to give suggestions